File: geeksforgeeks/linkedList/PushNode.cs
public class Node
{
    public int data;
    public Node next;
    public Node(int d){data=d;next=null;}

    public void push(int new_data)
    {
        Node new_node = new Node(new_data);
        new_node.next = head;
        head = new_node;
    }
}

File: geeksforgeeks/linkedList/pushNode.py
class Node:
    def __init__(self,data):
        self.data = data
        self.next = None
        
class LinkList:
    def __init__(self):
        self.head = None

class PushNode:
    def push(self, new_data):
        new_node = Node(new_data)
        new_node.next = self.head
        
        self.head = new_node